Date: Wed, 18 Aug 2004 21:18:19 +0200
From: Andreas Gruenbacher <agruen@suse.de>
Subject: Resolve duplicate/conflicting MODULE_LICENSE tags

* fs/affs/affs.ko, drivers/media/radio/miropcm20.ko have
  MODULE_LICENSE("GPL") twice.

* drivers/scsi/pcmcia/fdomain_cs.ko, drivers/scsi/pcmcia/aha152x_cs.ko
  have both MODULE_LICENSE("GPL") and MODULE_LICENSE("Dual MPL/GPL").
  The common denominator seems to be MODULE_LICENSE("GPL").

* drivers/net/ppp_mppe.ko has MODULE_LICENSE("BSD without advertisement
  clause"). This is generally considered a GPL compatible license,
  and should probably be added to license_is_gpl_compatible().

Signed-off-by: Andreas Gruenbacher <agruen@suse.de>

Index: linux-2.6.8/fs/affs/inode.c
===================================================================
--- linux-2.6.8.orig/fs/affs/inode.c
+++ linux-2.6.8/fs/affs/inode.c
@@ -427,4 +427,3 @@ err:
 	affs_unlock_link(inode);
 	goto done;
 }
-MODULE_LICENSE("GPL");
Index: linux-2.6.8/drivers/scsi/pcmcia/fdomain_stub.c
===================================================================
--- linux-2.6.8.orig/drivers/scsi/pcmcia/fdomain_stub.c
+++ linux-2.6.8/drivers/scsi/pcmcia/fdomain_stub.c
@@ -59,7 +59,6 @@
 
 MODULE_AUTHOR("David Hinds <dahinds@users.sourceforge.net>");
 MODULE_DESCRIPTION("Future Domain PCMCIA SCSI driver");
-MODULE_LICENSE("Dual MPL/GPL");
 
 /* Bit map of interrupts to choose from */
 static int irq_mask = 0xdeb8;
Index: linux-2.6.8/drivers/media/radio/miropcm20-radio.c
===================================================================
--- linux-2.6.8.orig/drivers/media/radio/miropcm20-radio.c
+++ linux-2.6.8/drivers/media/radio/miropcm20-radio.c
@@ -258,7 +258,6 @@ static int __init pcm20_init(void)
 
 MODULE_AUTHOR("Ruurd Reitsma");
 MODULE_DESCRIPTION("A driver for the Miro PCM20 radio card.");
-MODULE_LICENSE("GPL");
 
 static void __exit pcm20_cleanup(void)
 {
Index: linux-2.6.8/drivers/scsi/pcmcia/aha152x_stub.c
===================================================================
--- linux-2.6.8.orig/drivers/scsi/pcmcia/aha152x_stub.c
+++ linux-2.6.8/drivers/scsi/pcmcia/aha152x_stub.c
@@ -91,8 +91,6 @@ MODULE_PARM(synchronous, "i");
 MODULE_PARM(reset_delay, "i");
 MODULE_PARM(ext_trans, "i");
 
-MODULE_LICENSE("Dual MPL/GPL");
-
 /*====================================================================*/
 
 typedef struct scsi_info_t {
